pkg/spiffe/spiffe.go

```diff
@@ -0,0 +1,63 @@
+package spiffe
```

Comment:
I'd push to move pkg/spiffe/spiffe.go into cmd/in-toto/spiffe.go and not export the functions so we don't need to worry about supporting external users that import it. If it's used in multiple places, we could move to internal/spiffe/spiffe.go to limit to only our internal usage.

Reply:
Placed into internal directory.